crossdock/endtoend/server/mocks.go

```diff
@@ -0,0 +1,55 @@
+package server
+
+import (
+	"github.com/opentracing/opentracing-go"
+	"github.com/opentracing/opentracing-go/log"
+)
+
+type mockSpan struct {}
```

Comment:
I am not following why we need this. Don't we want to generate real traces? And for testing purposes we can still use a real Jaeger tracer, just configure it with in-memory reporter.